requirements/base.in

```diff
@@ -17,3 +17,6 @@ edx-rest-api-client
 lti-consumer-xblock===6.3.0
```

Comment:
do we need to get rid of this line? Won't there be two xblock library versions floating around with this?

Reply:
never mind, from the .txt results below clearly it works